app/Repositories/ServerRepository.php

```diff
@@ -708,13 +687,13 @@ public function updateStartup($id, array $data, $admin = false)
                     // Is the variable required?
                     // @TODO: is this even logical to perform this check?
                     if (isset($data[$variable->env_variable]) && empty($data[$variable->env_variable])) {
-                        if ($variable->required === 1) {
+                        if ($variable->required) {
                             throw new DisplayException('A required service option variable field (' . $variable->env_variable . ') was included in this request but was left blank.');
                         }
                     }
 
                     // Variable hidden and/or not user editable
-                    if (($variable->user_viewable === 0 || $variable->user_editable === 0) && ! $admin) {
+                    if ((! $variable->user_viewable || ! $variable->user_editable) && ! $admin) {
```

Comment:
Thats just the code standards we're using. Its kind odd, but does improve readability I think.